src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseInviteService.cs

```diff
@@ -48,18 +47,16 @@ List<Guid> releaseVersionIds
             {
                 var sanitisedEmail = email.Trim();
 
-                var user = await userRepository.FindByEmail(sanitisedEmail);
-                if (user == null)
-                {
-                    return await CreateNewUserContributorInvite(releaseVersionIds, sanitisedEmail, publication.Title);
-                }
-
-                return await CreateExistingUserContributorInvite(
-                    releaseVersionIds,
-                    user.Id,
-                    sanitisedEmail,
-                    publication.Title
-                );
+                var activeUser = await userRepository.FindActiveUserByEmail(sanitisedEmail);
+
+                return activeUser is null
+                    ? await CreateInactiveUserContributorInvite(releaseVersionIds, sanitisedEmail, publication.Title)
```

Comment:
You could argue that `email.Trim();` should be within `CreateInactiveUserContributorInvite` so that the trim isn't forgotten if there were multiple cases using it.

Also possibly within `FindActiveUserByEmail` if it makes any difference to that query (whether `'test@test.com  '` matches `'test@test.com'` might depend on the column type or our database collation setting).

Also see my other comment about `CreateActiveUserContributorInvite`, because together you might be able to remove 

```csharp
var sanitisedEmail = email.Trim();
```

from this method.